pytorch/serving/README.md

```diff
@@ -12,29 +12,73 @@ The [Torchserve Model Archiver](https://github.com/pytorch/serve/blob/master/mod
 
 Follow the instructions found in the link above depending on whether you are intending to archive a model or a workflow. Use the provided container rather than installing the archiver with the example command below:
 
+#### For CPU
```

Comment:
Like I said in a previous comment, this is fine for now, but ideally we would have a more descriptive header.

Reply:
modified the header